### PR TITLE
CNDE-1948

### PR DESCRIPTION
**Pull request by joshua-mills-cdw**
## Notes

Created a stored procedure for accessing user report permissions

## JIRA

- **Related story**: [CNDE-1948](https://cdc-nbs.atlassian.net/browse/CNDE-1948)

## Checklist

- [x] PR focuses on a single story.
- [ ] New unit tests added and ensured they pass.
- [ ] Service has been tested in local and it works as expected.
- [ ] Documentation has been updated for this code change (if needed).
- [ ] Code follows the Java Coding Conventions (https://www.oracle.com/java/technologies/javase/codeconventions-programmingpractices.html).

## Types of changes

What types of changes does this PR introduces?

- [ ] Bugfix
- [x] New feature
- [ ] Breaking change

## Testing

- [ ] Does this PR has >90% code coverage?
- [ ] Is the screenshot attached for code coverage?
- [ ] Does the `gradle build` pass in your local? 
- [ ] Is the `gradle build` logs attached?

### CODE REVIEW

File: liquibase-service/src/main/resources/db/odse/routines/022-sp_user_report_permissions-001.sql
Comment (sveselev) [inline liquibase-service/src/main/resources/db/odse/routines/022-sp_user_report_permissions-001.sql:1]: Please add a copy of this script to the consolidated scripts (as `017-sp_user_report_permissions.sql`, no `-001` postfix needed), under the `db/upgrade/odse/routines` folder.

Comment (joshua-mills-cdw) [reply liquibase-service/src/main/resources/db/odse/routines/022-sp_user_report_permissions-001.sql:1]: Done. Thanks!

**Review by upasanapattnaik-eq — APPROVED** [review]
LGTM!

**Comment by upasanapattnaik-eq** [comment]
As this is an ODSE sproc, who are the intended users for this sproc?

**Pull request merged**